XMLRequest/public/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <input id="field1" />
+    <input id="field2" />
+    <input type="submit" id="sb" />
+    <div id="chat"></div>
+
+    <script>
+      const xml = new XMLHttpRequest();
+      const submitButton = document.getElementById('sb');
+      const chat = document.getElementById('chat');
+      submitButton.onclick = () => {
+        const f1 = document.getElementById('field1').value;
```

Comment:
instead of field1, please use a better name. like`name`, `comment`, `age`, etc. Please apply change throughout your code.